samples/drivers/video/capture/boards/arduino_nicla_vision_stm32h747xx_m7.conf

```diff
@@ -0,0 +1,15 @@
+CONFIG_VIDEO_BUFFER_POOL_NUM_MAX=1
+CONFIG_VIDEO_BUFFER_POOL_SZ_MAX=160000
+CONFIG_DMA=y
+
+# Arduino Nicla Vision has easy access to the USB console
+# So use it to check the log messages
+CONFIG_USB_DEVICE_STACK=y
```

Comment:
Is that controller not supported by `device_next` yet? Just `device`?
Not that it's blocking though. Just one more thing to convert some day.

Reply:
I did not test it, the only thing different with Nicla USB is the fact that it uses an ULPI phy to achieve high speeds.

Since it worked out of the box with the legacy USB device stack, I just kept this as default.

Reply:
Just added, I was adding a unnecessary underscore before.